### PR TITLE
PR with suggestions - [Improvement]: Help streamline / simplify dependency review action README 

### PR DESCRIPTION
**Pull request by am-stead**
Linked to: "[[Improvement]: Help streamline / simplify dependency review action README#14071](https://github.com/github/docs-content/issues/14071)"

Suggestions made: 
- Added TOC for easier navigation
- Re-worked intro into an "Overview" section and a "Viewing your results" sub-section.
   - Added some info on where/how to find job logs / job summary. (Needs checking for accuracy)
- Re-structured content into new heading and sub-headings. 
  - Re-structured "Installation" and added mini TOC ("Installation (Standard)", "Installation (GitHub Enterprise Server)"
  - Re-configured "Configuration options" section into "Configuration" and added mini TOC to two sub-sections: "Configuration options" and "Configuration methods".
  - Merged existing table notes and considerations for configuration options into one "Notes" sections after the large table. 
  - Split "configuration methods" into two sub-sub-sections ("Option 1: Using inline configuration..." and "Option 2: Using an external configuration file")
  - Re-worked same content for clarity - into set procedural steps. 
- Kept final sections as they are (Outputs, Blocking pull requests [re-named section heading to something more explanative], Getting help, Contributing, Licenses)
- Updated GHES links to "enterprise-server@**latest**" (e.g.) `https://docs.github.com/en/enterprise-server@latest/admin/code-security/managing-github-advanced-security-for-your-enterprise/enabling-github-advanced-security-for-your-enterprise` (The existing links link out to GHES 3.8 (soon to be deprecated), so this will keep the links up to date.) 
- Added a link to docs for general info on dependency review (suggestion)

To do:
- For "Outputs" - what is outputs used for? I would like to add an intro line here, but I'm not sure what to put. Could you advise?

### CODE REVIEW

**Comment by am-stead** [comment]
:wave: @jonjanego 

This week I took a look at the README for dependency-review-action. I have made some suggestions for your consideration (summarised above). ✨

File: README.md [hunk 1]
```diff
@@ -86,81 +119,107 @@ Configure this action by either inlining these options in your workflow file, or
...
 
-## Blocking pull requests
+## Using dependency review action to block a pull request from being merged
 
-The Dependency Review GitHub Action check will only block a pull request from being merged if the repository owner has required the check to pass before merging. For more information, see the [documentation on protected branches](https://docs.github.com/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging).
+You can configure your repository to block a pull request from being merged if the pull request fails the dependency review action check. To do this, the repository owner must configure branch protection settings that require the check to pass before merging. For more information, see "[Require status checks before merging](https://docs.github.com/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging)" in GitHub Docs documentation.   
 
 ## Outputs
 
+[TODO: Insert overview line here - what does output refer to? What / how it is used?]
```
Comment (am-stead) [inline README.md:221]: It would be good to add some sort of intro line here? But I wasn't sure what to put...

**Review by jonjanego — CHANGES_REQUESTED** [review]
this is _AMAZING_ thank you @am-stead 

i have filled in the "TODO" content and added a formatting suggestion

File: README.md [hunk 2]
```diff
@@ -86,91 +119,117 @@ Configure this action by either inlining these options in your workflow file, or
...
 ## Outputs
 
+Dependency review action can create [outputs](https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs), so that data from its execution can be used by other jobs in a workflow.
+
 - `comment-content` is generated with the same content as would be present in a Dependency Review Action comment.
 - `dependency-changes` holds all dependency changes in a JSON format. The following outputs are subsets of `dependency-changes` filtered based on the configuration:
-- `vulnerable-changes` holds information about dependency changes with vulnerable dependencies in a JSON format.
+  - `vulnerable-changes` holds information about dependency changes with vulnerable dependencies in a JSON format.
 - `invalid-license-changes` holds information about invalid or non-compliant license dependency changes in a JSON format.
 - `denied-changes` holds information about denied dependency changes in a JSON format.
```
Comment (am-stead) [inline README.md:227]:
```suggestion
  - `vulnerable-changes` holds information about dependency changes with vulnerable dependencies in a JSON format.
  - `invalid-license-changes` holds information about invalid or non-compliant license dependency changes in a JSON format.
  - `denied-changes` holds information about denied dependency changes in a JSON format.
```

File: README.md [hunk 3]
```diff
@@ -86,81 +119,107 @@ Configure this action by either inlining these options in your workflow file, or
...
+You can configure your repository to block a pull request from being merged if the pull request fails the dependency review action check. To do this, the repository owner must configure branch protection settings that require the check to pass before merging. For more information, see "[Require status checks before merging](https://docs.github.com/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging)" in GitHub Docs documentation.   
 
 ## Outputs
 
+[TODO: Insert overview line here - what does output refer to? What / how it is used?]
+
 - `comment-content` is generated with the same content as would be present in a Dependency Review Action comment.
 - `dependency-changes` holds all dependency changes in a JSON format. The following outputs are subsets of `dependency-changes` filtered based on the configuration:
 - `vulnerable-changes` holds information about dependency changes with vulnerable dependencies in a JSON format.
```
[the remark this answers is not in the document]
Comment (am-stead) [reply README.md]: I've indented these now to match `vulnerable-changes`!

**Review by jonjanego — CHANGES_REQUESTED** [review]
one suggested removal, one comment.

thx!

File: README.md [hunk 4]
```diff
@@ -86,91 +119,117 @@ Configure this action by either inlining these options in your workflow file, or
...
-- `denied-changes` holds information about denied dependency changes in a JSON format.
+  - `vulnerable-changes` holds information about dependency changes with vulnerable dependencies in a JSON format.
+  - `invalid-license-changes` holds information about invalid or non-compliant license dependency changes in a JSON format.
+  - `denied-changes` holds information about denied dependency changes in a JSON format.
 
 > [!NOTE]
 > Action outputs are unicode strings [with a 1MB size limit](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#outputs-for-docker-container-and-javascript-actions).
 
-> [!IMPORTANT]
+> [!NOTE]
```
[the remark this answers is not in the document]
Comment (am-stead) [reply README.md]: I've pushed an update for this change now! 👍

**Review by jonjanego — APPROVED** [review]

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jonjanego) [reply README.md]: fantastic. merging!

**Review by TaZ1312 — APPROVED** [review]

**Pull request merged**